BasicSample/Assets/Rendering/Scripts/CubesGrid.cs

```diff
@@ -1,5 +1,6 @@
-using System.Collections;
```

Comment:
Is removing these headers intentional?

Reply:
Yep! The auto-formatter removed any unused `using` statements.

Reply:
cool, what auto-formatter are you using?

Reply:
Mostly the in-box Visual Studio one (I'm pretty sure, at least)! I tend to use the right-click menu in the Solution Explorer:

![image](https://user-images.githubusercontent.com/3580640/204397008-349e1717-9f54-44a0-9bca-6feeadc69a2d.png)


You can also do a single file with 
![image](https://user-images.githubusercontent.com/3580640/204396200-40be40a7-1cf6-41f0-8879-5639d1b75095.png), though I believe this option cleans up slightly fewer things by default.

Reply:
This is good to know, always been using VSCode and did not realize Visual Studio has this feature. Also, its been difficult to auto format C# code, this is great! Will add this to best practices documentation.